File: B面向对象/#复习最新课件/unit2 java Class Implementation(更新部分)/java-source/2.1.1/Java形参与实参的结合方式演示/Point2D.java

public class Point2D {
	
	private float x;
	private float y;
	
	public Point2D(float initialX, float initialY) {
		
		x = initialX;
		y = initialY;
	}
	
	public float getX() {
		
		return x;
	}
	
	public float getY() {
		
		return y;
	}	
	public void setX(float newX){
		
		x=newX;
	}
	public void setY(float newY){
		
		y=newY;
	}
	public String toString(){
		return getX()+","+getY();
	}
	
}

File: B面向对象/#复习最新课件/unit1 Object-oriented foundation and class diagram（更新）/source/1.2.3/Point2D.cpp
  #include "point2D.h"
  float Point2D::getX() const{
      return x;
  }
  float Point2D::getY() const{
	  return y;
  }
  //与ppt中构造函数演示相对应
  /* 1,3*/
  Point2D::Point2D(float initialX, float initialY) {

	  x = initialX;
	  y = initialY;
  }
  Point2D::~Point2D(){

  }

